climpred/graphics.py

```diff
@@ -150,29 +158,30 @@ def plot_bootstrapped_skill_over_leadyear(init_skill,
             label='uninitialized with ' + str(sig) + '% confidence interval')
         ax.axhline(y=uninit_skill, c='steelblue', ls=':')
     # persistence
-    if pers_skill is not None and pers_ci is not None:
-        ax.errorbar(
-            pers_skill.lead,
-            pers_skill,
-            yerr=[
-                pers_skill - pers_ci.isel(quantile=0),
-                pers_ci.isel(quantile=1) - pers_skill
-            ],
-            fmt='--o',
-            capsize=capsize,
-            c=c_pers,
-            label='persistence with ' + str(pers_sig) +
-            '% confidence interval')
-    if p_pers_over_init is not None:
-        for t in pers_skill.lead.values:
-            ax.text(
-                pers_skill.lead.sel(lead=t),
-                pers_ci.isel(quantile=0).sel(lead=t).values,
-                "%.2f" % float(p_pers_over_init.sel(lead=t).values),
-                horizontalalignment='center',
-                verticalalignment='bottom',
-                fontsize=fontsize,
-                color=c_pers)
+    if plot_persistence:
```

Comment:
I want to have that switch because I often dont have the persistence forecast to be shown